notebooks/COS/ViewData/ViewData.ipynb

```diff
@@ -0,0 +1,1452 @@
+    "- `matplotlib.pyplot` for plotting data\n",
+    "- `astroquery.mast Mast and Observations` for finding and downloading data from the [MAST](https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html) archive\n",
+    "\n",
+    "*Later on, we will import some functions from a local file*, for which we have imported the `sys` package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This line is just to make matplotlib look good in a notebook\n",
+    "%matplotlib inline\n",
+    "# Manipulating arrays\n",
+    "import numpy as np\n",
+    "# Our arrays will encounter divide by 0 warnings, which is OK, but we want to suppress the warning text with the next code line.\n",
+    "   # SO DO NOT COPY THIS NEXT LINE INTO YOUR CODE WITHOUT CAREFUL CONSIDERATION!\n",
+    "np.warnings.filterwarnings('ignore')\n",
+    "# Reading in data\n",
+    "from astropy.io import fits\n",
+    "from astropy.table import Table\n",
+    "# Dealing with units and plotting them\n",
+    "from astropy import units as u\n",
+    "from astropy.visualization import quantity_support\n",
+    "quantity_support()\n",
+    "# Plotting\n",
+    "import matplotlib.pyplot as plt\n",
+    "# Downloading data from archive\n",
+    "from astroquery.mast import Mast\n",
+    "from astroquery.mast import Observations\n",
+    "from astroquery.mast import Catalogs\n",
+    "\n",
+    "import sys # We need import to look in the directory above this one when we look for some functions later.\n",
+    "sys.path.append('../') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will also define a few directories in which to place our data and plots, as well as a few colors we will use in plots later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# These will be important directories for the notebook\n",
+    "cwd = !pwd\n",
+    "cwd = cwd[0]\n",
+    "\n",
+    "!mkdir ./data\n",
+    "!mkdir ./output/\n",
+    "!mkdir ./output/plots\n",
+    "\n",
+    "datadir = cwd + '/data/'\n",
+    "outputsdir = cwd + '/output/'\n",
+    "plotsdir = cwd + '/output/plots/'\n",
+    "\n",
+    "# Specifying a few colors for plotting by their hex code - feel free to change these, I just like them!\n",
+    "## Many search engines will show you the hex color - i.e. Google \"#BC8C5B\" to see this orange color\n",
+    "segment_colors = {'FUVA':'#BC8C5B', 'FUVB':'#4B6CA4', 'NUVA':'#1813CE','NUVB':'#61946E','NUVC':'#8C1A11'}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will be working with some FUV and NUV datasets downloaded in the cell below. \n",
+    "### Searching for and downloading data is out of the scope of this tutorial. If you wish to learn more, please see our [tutorial on downloading COS data](#NEEDSLINK!)"
```

Comment:
I think it's fine to do say this, but you might want to just write one sentence on what this data set is.